### PR TITLE
feat: Tag sampling rates + type on replay_event

### PR DESCRIPTION
**Pull request by billyvg**
Add tags for `sessionSampleRate`, `errorSampleRate`, and `replayType` (`error` or `sample`). Right now these are added as tags for debugging purposes.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1237,11 +1237,19 @@ export class Replay implements Integration {
           if (session) {
             client._updateSessionFromEvent(session, preparedEvent);
           }
+
           preparedEvent.sdk = {
             ...preparedEvent.sdk,
             ...sdkInfo,
           };
 
+          preparedEvent.tags = {
```
Comment (JoshFerge) [inline src/index.ts:1246]: we'll want to remove this before beta is over

**Review by JoshFerge — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (billyvg) [reply src/index.ts:1246]: It could be useful info for customers?

Comment (JoshFerge) [reply src/index.ts:1246]: tags are reserved for customer use only by other sdk standards (we should at some point remove our use of them as well)

Comment (billyvg) [reply src/index.ts:1246]: Ah that's true -- I suppose if people find this useful we can stick them in the envelope headers like Bruno suggested.

**Pull request merged**